src/containers/AgreementPage/components/AgreementForm.tsx

```diff
@@ -94,7 +94,7 @@ const AgreementForm = ({ onUpsert, agreement }: Props) => {
       title: values.title,
       content: values.content,
       copyright: {
-        license: licenses.find(license => license.license === values.license),
+        license: licenses!.find(license => license.license === values.license),
```

Comment:
Er det mulig å unngå `!` på licenses alle steder det brukes?

Reply:
Alternativet ville vært å legge på en `!` inne i definisjonen av useQueries, og returnere et tomt array hvis den ikke har blitt hentet ennå. Jeg er egentlig litt fan av det, i og med at den kun hentes en gang. Hvis du er enig kan jeg få lagt det inn.

Reply:
Men på en generell basis tillater ikke React-query dette. Du kan spesifisere både en verdi som brukes inntil den blir erstattet av faktisk data og en placeholder-verdi, men det blir for vanskelig å type useQuery basert på bruken av disse to. Det blir enda vanskeligere når man skal ta hensyn til variabler som `enabled`.

Reply:
Høres ut som en fin løsning for min del. Greit å redusere antallet `!` til bare én hvertfall

Reply:
Ser nå at dette blir vanskelig å få til. Vil tro det er mulig dersom vi overstyrer en god del av react-query sine type-definisjoner, men kan se for meg at dette vil bli mer jobb enn det er verdt med tanke på fremtidig vedlikehold.